CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Implementation of the RabbitMQ EventBus
 - DeadLetter APIs and memory implementation for storing events that failed delivery
 - RecipientRewriteTable Aliases and associated WebAdmin routes
+- Dead letter reDeliver routes on top of WebAdmin
+- Dead letter Cassandra implementation
+- Deleted Message vault implementation on top of MailReposiotries
+- WebAdmin routes for restoring deleted messages
```

Comment:
Do we have a page talking about the Deleted Messages Vault somewhere? Because we probably need to provide a high level view of the feature as a link.

Reply:
Documentation is planned later on